### DISCUSSION TITLE
Multiprocessing on Windows

### DISCUSSION THREAD

**Discussion by cora-const** [vortex-exoplanet/VIP#627] [Q&A]
Hi!
I’m trying to familiarize myself with VIP on a Windows machine, but I am running into some multiprocessing issues while working though the tutorials. For example, when trying to run `snrmap1 = snrmap(fr_pca1, fwhm_naco, plot=True)` from tutorial 1, I get `ValueError: cannot find context for 'fork'`

I know `fork()` isn’t supported on Windows; is there a way to get around this without having to install a new OS?

**Comment by VChristiaens** [comment]
Hi @cora-const, upon asking some VIP users on Windows machines, what has worked for them is WSL: https://learn.microsoft.com/en-us/windows/wsl/install
Apparently it solves all compatibility problems, while allowing you to keep Windows in the background. The first steps at the following link may also be useful for you to get miniconda properly installed with WSL, before installing VIP: https://awhug.github.io/2020-07-10-installing-pymc3-and-theano-on-windows-10-with-wsl/
Cheers
